src/elastio-snap.c

```diff
@@ -3857,6 +3976,10 @@ static void __on_bio_read_complete(struct bio *bio, int err){
 		goto error;
 	}
 
+	if (elastio_snap_bio_op_flagged(bio, REQ_DRV)) {
+		bio_traced = true;
```

Comment:
Flags are being zeroed in the function below. I didn't want to change that function so added this bool value to restore the flag in the end.